crates/swift-bridge-macro/tests/ui/args-into-argument-not-found.rs

```diff
@@ -0,0 +1,18 @@
+//! # To Run
+//! cargo test -p swift-bridge-macro -- ui trybuild=args-into-argument-not-found.rs
+
+#[swift_bridge::bridge]
+mod ffi {
+    extern "Rust" {
+        #[swift_bridge(args_into = (arg, arg_typo))]
+        fn some_function(arg: u8);
+    }
```

Comment:
Can we add a second extern "Rust" block here that uses a method?

Maybe something like:

```rust
extern "Rust" {
    type SomeType;

    #[swift_bridge(args_into = (arg, arg_typo))]
    fn some_method(&self, arg: u8);
}
```